plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveTableProperties.java

```diff
@@ -142,6 +146,23 @@ public HiveTableProperties(
                         "ORC Bloom filter false positive probability",
                         orcWriterConfig.getDefaultBloomFilterFpp(),
                         false),
+                new PropertyMetadata<>(
+                        PARQUET_BLOOM_FILTER_COLUMNS,
+                        "Parquet Bloom filter index columns",
+                        new ArrayType(VARCHAR),
+                        List.class,
+                        ImmutableList.of(),
+                        false,
+                        value -> ((List<?>) value).stream()
+                                .map(String.class::cast)
+                                .map(name -> name.toLowerCase(ENGLISH))
+                                .collect(toImmutableList()),
+                        value -> value),
+                doubleProperty(
+                        PARQUET_BLOOM_FILTER_FPP,
+                        "Parquet Bloom filter false positive probability",
+                        parquetWriterConfig.getBloomFilterFpp(),
```

Comment:
Add a value validation here

Reply:
Will add validation in `ParquetWriterConfig#setBloomFilterFpp`.

Reply:
Config is one thing. But user can pass an incorrect value to the session property and should get the error instantly instead of waiting for query to run

Reply:
Ah, okay. Will add a check here as well to fail fast on invalid fpp values originating from session.